benchmark/vochain_benchmark_test.go

```diff
@@ -115,17 +115,16 @@ func BenchmarkVochain(b *testing.B) {
 		Process: processData,
 	}
 
-	txBytes, err := proto.Marshal(process)
+	stx := &models.SignedTx{}
+	stx.Tx, err = proto.Marshal(&models.Tx{Payload: &models.Tx_NewProcess{NewProcess: process}})
```

Comment:
Since stx is declared the line before, would not be more idiomatic to use `if stx.Tx, err := ...; err != nil {...}` ?

Reply:
in this case you can't, because stx.Tx is not a variable name, and `:=` can only be used to define, not assign

Reply:
you cannot do 
```	stx := &models.SignedTx{}
	if stx.Tx, err = proto.Marshal(&models.Tx{Payload: &models.Tx_NewProcess{NewProcess: process}}); err != nil {
		b.Fatal("cannot marshal process")
	}
	if stx.Signature, err = dvoteServer.Signer.Sign(stx.Tx); err != nil {
		b.Fatalf("cannot sign oracle tx: %s", err)
	}
```
??

Reply:
sure, you can, I'd probably do it that way. I was just replying to your earlier use of `:=` :)

Reply:
ahh oki, yeah thats why I edited the comment (2 seconds after) :)